lua/plenary/functional.lua

```diff
@@ -38,4 +38,21 @@ f.all = function(f, iterable)
   return true
 end
 
+function f.make_selector(n)
```

Comment:
We do want to expose this, right? And is the name fine?

Reply:
`select_only` I think would be better.